### ISSUE TITLE
Accessibility Issues with Contact Form Element

### ISSUE THREAD

**Issue by clicksports** [KriesiMedia/Enfold-Feature-Requests#115]
Hello,

since the European Accessibility Act ist basically right around the corner, we are preparing a lot of our customer sites running Enfold to meet the new standards these days. One of our checking software found some issues with the default contact form that comes with the Enfold theme.

The reported issues are as follows:

Guideline: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Fieldset.Name
Message: This fieldset element does not have a name available to an accessibility API. Valid names are: legend element, aria-label undefined, aria-labelledby undefined.
Code (abbreviated): <fieldset><p class=” first_form form_ele…</fieldset>

—

Guideline: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputText.Name
Message: This textinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.
Code: <input type=”text” name=”avia_6_1″ class=”hidden ” id=”avia_6_1″ value=””>

—

Guideline: WCAG2AA.Principle1.Guideline1_3.1_3_1.H71.NoLegend
Message: Fieldset does not contain a legend element. All fieldsets should contain a legend element that describes a description of the field group.
Code: (abbreviated): <fieldset><p class=” first_form form_ele…</fieldset>

—

Maybe you guys can have a look at the reported issues and check if you can release an update with the coming versions.
